schemas/twitter.js

```diff
@@ -0,0 +1,167 @@
+import * as twitter from '../apis/twitter';
+
+import {
+    GraphQLSchema,
+    GraphQLObjectType,
+    GraphQLID,
+    GraphQLString,
+    GraphQLNonNull,
+    GraphQLInt,
+    GraphQLList,
+} from 'graphql';
+
+let UserType = new GraphQLObjectType({
+
+    name: 'TwitterUser',
+    description: 'Twitter user, or as we like to say in France: a \'tweetos\'',
+    fields: () => ({
+        created_at: { type: GraphQLString },
```

Comment:
I'll try my best to keep consistency with your apis (even if i'm not really comfortable with this :p)